### PR TITLE
Docs 8911 port istio 1.6 changes pa

### PR DESCRIPTION
**Pull request by priyaayyar**

### CODE REVIEW

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 1]
```diff
@@ -81,16 +81,83 @@ In your network, you might need to whitelist the hostnames and ports of external
 |===
 
 [[install_Istio]]
-== Installing Istio
+== Download, Install, and Configure Istio
 
-You can install Istio 1.4.x or 1.5.x in your environment to support Anypoint Service Mesh. However, the Istio configuration for Anypoint Service Mesh differs based on the Istio version that you installed.
+You can install Istio 1.6.x, 1.5.x, or 1.4.x in your environment to support Anypoint Service Mesh. However, the Istio configuration for Anypoint Service Mesh differs based on the Istio version that you installed.
```
Comment (tbruno) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]:
```suggestion
You can install Istio 1.6.x in your environment to support Anypoint Service Mesh. However, the Istio configuration for Anypoint Service Mesh differs based on the Istio version that you installed.
```

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 2]
```diff
@@ -81,16 +81,83 @@ In your network, you might need to whitelist the hostnames and ports of external
...
+----  
+When using the full manifest, ensure that you choose the correct profile. In the example, the `default` profile is used. 
+
+To install the manifest, run:
++
+`istioctl manifest apply -f <manifest-file.yaml>`
+
+//So...doesn't configuration begin with specifying the correct profile? Profile is the first component of the install process, so it should be described first, shouldn't it?
+
+=== Install and Configure Istio 1.5.x
```
Comment (tbruno) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: We need to delete this section as Istio 1.5 is not supported in 1.1

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 3]
```diff
@@ -156,12 +223,11 @@ To install the manifest, run:
 +
 `istioctl manifest apply -f <manifest-file.yaml>`
 
-=== Install Istio 1.4.x
+=== Install and Configure Istio 1.4.x
```
Comment (tbruno) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: We need to delete this section as Istio 1.4 is not supported in 1.1

**Review by tbruno — COMMENTED** [review]
Thanks.

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 1 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 2 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 3 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Done! Thanks!

**Pull request merged**